js/src/Map.js

```diff
@@ -56,7 +56,7 @@ var LeafletMapModel = widgets.DOMWidgetModel.extend({
         inertia_deceleration : 3000,
         inertia_max_speed : 1500,
         // inertia_threshold : int(?)
-        zoom_control : true,
+        zoom_control : false,
```

Comment:
Actually, this line should be completely removed, no need to synchronize the `zoom_control` value with the front-end, only the Python side needs to know about this attribute.